tests/provision/provisioning_test.go

```diff
@@ -107,11 +107,12 @@ var _ = Describe("{ReplicaAffinity}", func() {
 			for _, ctx := range contexts {
 				TearDownContext(ctx, opts)
 			}
+
+			JustAfterEach(func() {
+				AfterEachTest(contexts)
+			})
 		}
 	})
-	JustAfterEach(func() {
```

Comment:
don't move this code, this has to be outside `It()`

Reply:
Made changes as suggested